### ISSUE TITLE
[Node][21.luis-with-appinsights]: Sample has invalid botfile and doesn't run when corrected

### ISSUE THREAD

**Issue by andybeak** [microsoft/BotBuilder-Samples#1089]
## Sample information
1. Sample type: Sample
2. Sample language: nodejs
3. Sample name: 21.luis-with-appinsights/

## Describe the bug
On line 34 of the supplied bot file there is a comma which causes the JSON to fail validation and cause the bot to complain that there is no valid bot file.

If you fix that then you get a new error 
```
/home/andrew/projects/bot/index.js:59
    applicationId: luisConfig.appId,                             
TypeError: Cannot read property 'appId' of null
```

Which indicates that this line is returning null:
```
const luisConfig = botConfig.findServiceByNameOrId(LUIS_CONFIGURATION);
```

## To Reproduce
1. Install fresh project and run npm install
2. npm start and observe that there is a report about no botfile
3. remove the superfluous comma so that the botfile parses cleanly
4. npm start and observe that luisConfig is null

## Expected behavior
I would expect that luisConfig is populated with the values specified in the bot file

## Screenshots
Not applicable

## Additional context
$ nodejs --version
v11.3.0
$ npm --version
6.4.1
$ uname -r
4.15.0-39-generic

[bug]

**Comment by stevengum** [comment]
@andybeak, did you follow the readme or create a LUIS app and connect it to your .bot file? If so, the code in the `index.js` needs to be updated with the name of the LUIS App's item in the .bot file. 

Regarding the superfluous comma, it was removed in this [commit](https://github.com/Microsoft/BotBuilder-Samples/blob/c72cfba161d962672acf40f28f1b1259fff60f79/samples/javascript_nodejs/21.luis-with-appinsights/luis-with-appinsights.bot).

**Comment by andybeak** [comment]
I followed the readme.  There was a mismatch between the index and the botfile.

I submitted a pull request (https://github.com/Microsoft/BotBuilder-Samples/pull/1090/commits/fb4907c398199c9006a88efa748dfbcbd81b9a6a) but was told that hardcoding the name into the config was a bad idea.

In any event the live code appears to have the change I suggested (https://github.com/Microsoft/BotBuilder-Samples/blob/02010dc608fd325338cf43e27480ef27a1851690/samples/javascript_nodejs/21.luis-with-appinsights/luis-with-appinsights.bot#L2).

This issue can be closed.
